exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -127,19 +187,33 @@ void createTestKitWithBuilderForMultipleDifferentServicesVarargs() {
   void createTestKitWithTimeService() {
     TimeProvider timeProvider = FakeTimeProvider.create(TIME);
     try (TestKit testKit = TestKit.builder()
-        .withService(TestServiceModule.class)
-        .withTimeService(timeProvider)
+        .withDeployedArtifact(ARTIFACT_ID, ARTIFACT_FILENAME)
+        .withService(ARTIFACT_ID, SERVICE_NAME, SERVICE_ID)
+        .withTimeService(TIME_SERVICE_NAME, TIME_SERVICE_ID, timeProvider)
         .build()) {
-      TestService service = testKit.getService(TestService.SERVICE_ID, TestService.class);
+      TestService service = testKit.getService(SERVICE_NAME, TestService.class);
       checkTestServiceInitialization(testKit, service);
     }
   }
 
-  private void checkTestServiceInitialization(TestKit testKit, TestService service) {
-    // Check that TestKit contains an instance of TestService
-    assertThat(service.getId()).isEqualTo(TestService.SERVICE_ID);
-    assertThat(service.getName()).isEqualTo(TestService.SERVICE_NAME);
+  @Test
+  void createTestKitWithSeveralTimeServices() {
+    String timeServiceName2 = "Time service 2";
+    int timeServiceId2 = 18;
+    TimeProvider timeProvider = FakeTimeProvider.create(TIME);
+    TimeProvider timeProvider2 = FakeTimeProvider.create(TIME.plusDays(1));
+    try (TestKit testKit = TestKit.builder()
+        .withDeployedArtifact(ARTIFACT_ID, ARTIFACT_FILENAME)
+        .withService(ARTIFACT_ID, SERVICE_NAME, SERVICE_ID)
+        .withTimeService(TIME_SERVICE_NAME, TIME_SERVICE_ID, timeProvider)
+        .withTimeService(timeServiceName2, timeServiceId2, timeProvider2)
+        .build()) {
+      TestService service = testKit.getService(SERVICE_NAME, TestService.class);
+      checkTestServiceInitialization(testKit, service);
+    }
+  }
 
+  private void checkTestServiceInitialization(TestKit testKit, TestService service) {
```

Comment:
This code and checkTestService2Initialization do not check that the services with the given parameters are started. It can probably be done with the same approach as in time schema: #1136